features/feed.feature

```diff
@@ -120,3 +120,12 @@ Feature: Feed
     And post "1" in the feed is "Article2"
     And post "2" in the feed is "Article1"
     And the feed does not contain "Note1"
+
+  Scenario: Feed is sanitised
```

Comment:
Do we also want to add a test case for a reply thread? (although functionality should be covered by the post entity)